packages/delegate/src/getFieldsNotInSubschema.ts

```diff
@@ -5,7 +5,7 @@ import { Maybe } from '@graphql-tools/utils';
 import { isSubschemaConfig } from './subschemaConfig';
 import { MergedTypeInfo, SubschemaConfig, StitchingInfo } from './types';
 import { memoizeInfoAnd2Objects } from './memoize';
-import { collectFields, ExecutionContext } from 'graphql/execution/execute.js';
+import { collectFields, ExecutionContext } from 'graphql/execution/execute';
```

Comment:
This breaks NodeJS ESM  which I already tried before :)

Reply:
Ah dayum ):

Reply:
Maybe we can fix it in bundler level(bob) but not sure.

Reply:
Would a poor man's find/replace after bundling suffice? Like search for `.js` imports and replace with `.mjs`?

Reply:
What if we avoided these deep imports from graphql js….. any other .js imports?